src/IECoreArnold/Renderer.cpp

```diff
@@ -595,6 +595,14 @@ class ArnoldOutput : public IECore::RefCounted
 					}
 				}
 			}
+
+			// Decide if this render should be updated at interactive rates or
+			// not. We update all beauty outputs interactively by default, and
+			// allow others to be overridden using a parameter.
+			m_updateInteractively = parameter<bool>(
+				output->parameters(), "updateInteractively",
+				boost::starts_with( m_data, "RGBA " ) || boost::starts_with( m_data, "RGB " )
```

Comment:
This logic hasn't really changed, but I do wonder whether there could ever be a situation where there are multiple aovs matching this, producing unnecessary updates.  I guess there really shouldn't be ... the only reason I could think of for having a second RGBA aov would be if you were rendering deep beauty, and no one should be doing that interactively

Reply:
In that unlikely event (we can't render deep to the Catalogue anyway), the new `updateInteractively` parameter could be used to tell Arnold which of your multiple beauty's you care about.